### PR TITLE
feat(discordx): add priority sort on events

### PR DESCRIPTION
**Pull request by oceanroleplay**
**Please describe the changes this PR makes and why it should be merged:**

add priority on events

## Package

discordx

### CODE REVIEW

File: packages/discordx/examples/event/commands/priority.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { Discord, On } from "../../../src/index.js";
+
+enum PriorityLevel {
```
Comment (VictoriqueMoe) [inline packages/discordx/examples/event/commands/priority.ts:3]: wrong way around. 
LOW is actually 0, this is the highest priority

**Review by VictoriqueMoe — CHANGES_REQUESTED** [review]

**Review by VictoriqueMoe — APPROVED** [review]

**Review by VictoriqueMoe — APPROVED** [review]

File: packages/discordx/src/Client.ts [hunk 2]
```diff
@@ -1201,8 +1201,9 @@ export class Client extends ClientJS {
 
     const matchedPrefix = prefixRegex.exec(message.content)?.at(1) ?? "unknown";
     const isPrefixBaseCommand = mappedPrefix.includes(matchedPrefix);
-    const contentWithoutPrefix =
-      message.content.replace(prefixRegex, "").trim() + " ";
+    const contentWithoutPrefix = `${message.content
```
Comment (VictoriqueMoe) [inline packages/discordx/src/Client.ts:1204]: why?

Comment (VictoriqueMoe) [reply packages/discordx/src/Client.ts:1204]: this just needs to be `message.content.replace(prefixRegex, "").trim()`

**Review by VictoriqueMoe — CHANGES_REQUESTED** [review]

File: packages/discordx/src/Client.ts (hunk 2 above)
Comment (VictoriqueMoe) [reply packages/discordx/src/Client.ts:1204]: no need to wrap it in a template string

Comment (oceanroleplay) [reply packages/discordx/src/Client.ts:1204]: there is a space at the end

Comment (VictoriqueMoe) [reply packages/discordx/src/Client.ts:1204]: just noticed, sorry

**Review by VictoriqueMoe — APPROVED** [review]

**Pull request merged**